docker/package/defaults/tezos-baker.conf

```diff
@@ -5,5 +5,6 @@
 
 # shellcheck disable=SC2034
 DATA_DIR="/var/lib/tezos/baker"
```

Comment:
Btw, is it ok that both baker and node use `DATA_DIR` variable (for different purposes, with different values), are they isolated from each other?

Reply:
yes, as I said earlier this environment variables are used only in the scope of running service executables